packages/common/src/lib/format-appeal-documents.js

```diff
@@ -21,10 +34,39 @@ exports.formatNewDescription = (caseData) => {
 };
 
 /**
- *
+ * @param {import("../client/appeals-api-client").AppealCaseDetailed} caseData
+ * @returns {boolean}
+ */
+const hasNotificationMethods = (caseData) =>
+	!!caseData.AppealCaseLpaNotificationMethod && !!caseData.AppealCaseLpaNotificationMethod.length;
+exports.hasNotificationMethods = hasNotificationMethods;
+
+/**
+ * @param {import("../client/appeals-api-client").AppealCaseDetailed} caseData
+ * @returns {string|undefined}
+ */
+exports.formatNotificationMethod = (caseData) => {
+	if (!hasNotificationMethods(caseData)) {
```

Comment:
oh is the funny exports line so that you can use the function internally here? I think you can just reference it as `exports.hasNotificationMethods`

Reply:
it was indeed, will update it